### PR TITLE
Preloaded icons are better now

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Mark spotted some issues with my preloaded icons so this PR addresses those. Thanks Mark!

### CODE REVIEW

File: src/ui/components/Header/UserOptions.tsx [hunk 1]
```diff
@@ -50,39 +50,38 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
...
-        <button className="row group" onClick={onSettingsClick}>
-          <Icon filename="settings" className="bg-iconColor" />
-          <span>Settings</span>
-        </button>
-        {window.__IS_RECORD_REPLAY_RUNTIME__ || noBrowserItem ? null : (
-          <button className="row group" onClick={onLaunchClick}>
-            <Icon filename="replay-logo" className="bg-iconColor" />
-            <span>Launch Replay</span>
+    <>
+      {preloadIcons}
```
Comment (bvaughn) [inline src/ui/components/Header/UserOptions.tsx:55]: There's no reason for this to be a separate variable, is there? Why not just
```jsx
return (
  <>
    <div style={{ display: "none" }}>
      <Icon filename="docs" className="bg-iconColor" />
      <Icon filename="help" className="bg-iconColor" />
      <Icon filename="settings" className="bg-iconColor" />
      <Icon filename="replay-logo" className="bg-iconColor" />
    </div>
    <div className="user-options text-blue-400">
      <Dropdown
        buttonContent={<MaterialIcon iconSize="xl">more_horiz</MaterialIcon>}
        setExpanded={setExpanded}
        expanded={expanded}
        orientation="bottom"
      >
        <button className="row group" onClick={onDocsClick}>
          <Icon filename="docs" className="bg-iconColor" />
          <span>Docs</span>
        </button>
        <ExternalLink
          className="row group"
          href="https://discord.gg/n2dTK6kcRX"
        >
          <Icon filename="help" className="bg-iconColor" />
          <span>Chat with us</span>
        </ExternalLink>
        <button className="row group" onClick={onSettingsClick}>
          <Icon filename="settings" className="bg-iconColor" />
          <span>Settings</span>
        </button>
        {window.__IS_RECORD_REPLAY_RUNTIME__ || noBrowserItem ? null : (
          <button className="row group" onClick={onLaunchClick}>
            <Icon filename="replay-logo" className="bg-iconColor" />
            <span>Launch Replay</span>
          </button>
        )}
        <LoginButton />
      </Dropdown>
    </div>
  </>
);
```

Comment (jonbell-lot23) [reply src/ui/components/Header/UserOptions.tsx:55]: Good point. When I had done this in the past, I saw flickering behaviour. And when I did it this way, the flicker went away.

But clearly the flicker had nothing to do with whether or not I used a variable, and I just was being superstitious. 


Will address. Thank you for helping me improve!

Comment (jonbell-lot23) [reply src/ui/components/Header/UserOptions.tsx:55]: I did, sorry about that. Filed a new ticket: https://linear.app/replay/issue/DES-477/address-pr-feedback-in-8989

Comment (jonbell-lot23) [reply src/ui/components/Header/UserOptions.tsx:55]: New PR: https://github.com/replayio/devtools/pull/9005

**Pull request merged**